src/styles/common/_grid.scss

```diff
@@ -0,0 +1,54 @@
+$gap: 1rem;
+$max-width: 100ch;
+$min-side-margin: 1rem;
+$grid-gap: 1em;
+$separator-width: 2px;
+$separator-offset: 5px;
+$n-columns: 12;
+
+/* uniprot-website grid system */
+.uniprot-grid {
+  display: grid;
+  grid-template-columns:
+        // left-margin
+
+    minmax($min-side-margin, calc((100% - #{$max-width}) / 2))
+    // main grid columns
+    repeat(12, 1fr)
+    // right-margin
+    minmax($min-side-margin, calc((100% - #{$max-width}) / 2));
+
+  &--with-background > * {
+    background: salmon;
```

Comment:
I guess this is not part of the grid and should be handled separately?

Reply:
Where do you think is a better place to put it?